stl/inc/ranges

```diff
@@ -728,6 +735,482 @@ namespace ranges {
         using all_t = decltype(all(_STD declval<_Rng>()));
     } // namespace views
 
+    // CLASS TEMPLATE ranges::transform_view
+    template <class _Rng, class _Fn> // TRANSITION, LLVM-47414
+    concept _Meow = range<const _Rng>&& regular_invocable<const _Fn&, range_reference_t<const _Rng>>;

```

Comment:
Should we ship a cat noise identifier in production code? Even if it is a compiler bug workaround, and even if we do love cats, perhaps this name should be more descriptive? :cat: :cat2:

(I am actually okay with "let's meow in production code and be legends" after seeing the usage below, where this just wraps the two concepts.)

Reply:
I can call this `_Can_const_transform` - we should save `_Meow` for something truly epic.